src/routes/events.ts

```diff
@@ -30,6 +30,28 @@ export default async function events(fastify: FastifyInstance) {
       });
     };
 
+    const updateNavBadge = async () => {
```

Comment:
directive updates the directive, this nav bar is outside the directive

Reply:
will try move this inside directive and see

Reply:
oh i see so this updates the count on the navbar. You can just remove the count from the navbar then, no need to duplicate what's already on the directive

Reply:
This might be trickier than i thought. While the filters work, i find them a bit confusing as they represent the state of both lists. In the screenshot below, all the 4 invalid items are on 1 tab so if i apply the invalid filter when on the other tab I get an empty list and have to manually switch tabs to view the invalid items. 

![Screenshot 2024-06-05 at 07-27-38 CHT User Management Tool](https://github.com/medic/cht-user-management/assets/10602047/2cc7661e-ad5c-4f96-94e8-de317bd2f66d)

I think we should move the filters closer to the table, it would be great if they were directly on the status column. The app only needs to show the user some sort of hint when there are invalid/failed items and these hints can have links that just auto-apply a filter

![Screenshot 2024-06-05 at 07-40-37 CHT User Management Tool](https://github.com/medic/cht-user-management/assets/10602047/fbfc6642-7ca2-46bc-9ba6-46a9900b3d6e)

Reply:
i see no problem showing 0 rows if the filter is applied since as long as we clearly show that there is a filter

Reply:
OK, we can change this behavior later then.